src/main/resources/static/css/home.css

```diff
@@ -0,0 +1,177 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Lato', sans-serif;
+}
+
+#main_container {
+    display: flex;
+    background-image: url("../img/home.png");
+    background-repeat: no-repeat;
+    background-size: 101% 100%;
+}
+
+    #most_recent_q {
```

Comment:
Can we do some code cleanup here? It would be great if we can have more consistent formatting of code.

Reply:
The purpose of the indentation here is to show the hierarchy of the elements